src/jit/emitarm64.cpp

```diff
@@ -2864,6 +2864,8 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
         case EA_8BYTE:
             result = INS_OPTS_1D;
             break;
+        default:
```

Comment:
Agreed.  I left it as default: break; for now to keep things functionally the same during bring up.